addons/block_code/ui/blocks/parameter_block/parameter_block.gd

```diff
@@ -15,6 +15,18 @@ var spawned_by: ParameterOutput
 func _ready():
 	super()
 	_background.color = color
+	_update_block_shape()
+
+
+func _on_definition_changed():
+	super()
+	_update_block_shape()
```

Comment:
The **_background** accessed in `_update_block_shape()` is only available when the node has been ready.  However, seems like `_on_definition_changed()` will be invoked by some where before this node becomes ready and shows error:
```
res://addons/block_code/ui/blocks/parameter_block/parameter_block.gd:29 - Invalid assignment of property or key 'is_pointy_value' with value of type 'bool' on a base object of type 'Nil'.
```
I think that here should add a if condition checking [is_node_ready()](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-is-node-ready) before `_update_block_shape()`.

Reply:
Good catch @starnight ! I appended a fixup commit. Adding `await ready` did the trick.